src/antsibull/data/ansible-community.py.j2

```diff
@@ -1,3 +1,7 @@
+{#
+  SPDX-License-Identifier: GPL-3.0-or-later
+  SPDX-FileCopyrightText: Ansible Project, 2020
+#}
```

Comment:
I was first a bit sceptical, but then I think for most templates this is a good idea. I didn't add it for some of the Debian files (control.j2 and changelog.j2). Fixed in ca8f7d7ef9c21780a908e85f42fac41170fd805e.

Reply:
I was first a bit sceptical, but then I think for most templates this is a good idea. I didn't add it for some of the Debian files (control.j2 and changelog.j2). Fixed in ca8f7d7ef9c21780a908e85f42fac41170fd805e.

Reply:
For changelog.j2 I have no idea how to add comments. For control.j2 I'm not sure... since it allows comments, I guess adding it would be ok. Let me do that...

Reply:
Implemented in ea0764e671dc2cda5777ce0a4afbfac18fce10a3.

Reply:
> For changelog.j2 I have no idea how to add comments.

`dpkg-parsechangelog` excludes lines starting with `#` from the generated changelog.

Reply:
Ok, I changed that in abe8058841e14cd1022c8e8d94038e68786ff670 as well.